build.sh

```diff
@@ -0,0 +1,5 @@
+set -x
+npm install
+npm start &
+./node_modules/.bin/wait-on http://localhost:3000
```

Comment:
This file should be divided into build.sh and generate-static-page.sh

Reply:
Or the entire app code could be rewritten but that's beyond my humble capabilities.